jetty-webapp/src/main/java/org/eclipse/jetty/webapp/WebAppContext.java

```diff
@@ -997,6 +1003,18 @@ public void addConfiguration(Configuration... configuration)
         return null;
     }
 
+    public void removeConfiguration(Configuration... configurations)
+    {
+        loadConfigurations();
```

Comment:
Seems weird to first call load before calling remove ... but I guess if you don't it might not have been loaded yet.